bsd/forms.py

```diff
@@ -20,7 +20,8 @@ class BSDEventForm(forms.ModelForm):
         label="Capacity Limit",
         min_value=0,
     )
-    duration_count = forms.IntegerField(min_value=0)
+    """Duration form field should be required and positive"""
```

Comment:
Didn't we determine BSD didn't require duration?

Reply:
Upon further testings, the api call fails if duration field is missing, and the admin UI seems to actually set a default value of duration field is missing. Also setting this value to 0 has weird result where start time doesn't display for event, so I removed that option too.

Reply:
Gotcha. Classic BSD, thanks for explaining.